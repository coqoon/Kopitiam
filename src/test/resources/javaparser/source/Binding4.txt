class Foo {
  int a;
  int bar () {
    return a + a;
  }
  void foo () {
    int b = a;
    int c = a + bar();
  }
}
