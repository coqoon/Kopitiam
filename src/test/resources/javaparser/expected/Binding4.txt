class Foo {
  int a;
  int bar () {
    int tmp_1 = this.a;
    int tmp_2 = this.a;
    return tmp_2 + tmp_1;
  }
  void foo () {
    int b = this.a;
    int tmp_2 = this.bar();
    int tmp_3 = this.a;
    int c = tmp_3 + tmp_2;
  }
}
